### PR TITLE
Move Input/Output to new commandio module

### PR DESCRIPTION
**Pull request by t-wissmann**
The types module should provide the core type definitions and this has
nothing to do with the types Input and Output which are used in the
interface of commands.

### CODE REVIEW

File: src/utils.h [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include <sstream>
 #include <string>
 
+#include "commandio.h"
 #include "types.h"
```
Comment (blarz) [inline src/utils.h:13]: Is this still needed?

File: src/settings.h [hunk 2]
```diff
@@ -4,6 +4,7 @@
 #include <string>
 
 #include "attribute_.h"
+#include "commandio.h"
 #include "globals.h"
 #include "object.h"
 #include "types.h"
```
Comment (blarz) [inline src/settings.h:10]: Is this still needed?

**Review by blarz — CHANGES_REQUESTED** [review]
In some places types.h include could probably be removed. Not sure why IWYU doesn't complain

**Comment by t-wissmann** [comment]
I think you're right! I've removed both includes and resolved the merge conflicts.

File: src/settings.h (hunk 2 above)
Comment (blarz) [reply src/settings.h]: Thanks!

**Review by blarz — APPROVED** [review]

**Pull request merged**